Aplib.Core.Tests/Collections/ExposedQueueTests.cs

```diff
@@ -33,7 +33,9 @@ public void ExposedQueue_WhenInitialized_ShouldHaveCorrectMaxCountAndCount()
     public void ExposedQueue_WhenInitializedWithArray_ShouldHaveCorrectMaxCountAndCount(int[] array, int? countParam, int expectedMaxCount, int expectedCount)
     {
         // Arrange
-        ExposedQueue<int> queue = new(array, countParam);
+        ExposedQueue<int> queue = countParam.HasValue
+            ? new ExposedQueue<int>(array, (int)countParam)
+            : new ExposedQueue<int>(array);
```

Comment:
Having conditionals in tests is considered a [test smell](https://testsmells.org/pages/testsmells.html#ConditionalTestLogic). 

You should provide different tests with or without the countparam (this is as easy as just copying the test, changing the name and the params)